mqt/ddsim/bindings.cpp

```diff
@@ -107,19 +107,20 @@ void dump_tensor_network(const py::object& circ, const std::string& filename) {
     py::object QuantumCircuit       = py::module::import("qiskit").attr("QuantumCircuit");
     py::object pyQasmQobjExperiment = py::module::import("qiskit.qobj").attr("QasmQobjExperiment");
 
+    std::unique_ptr<qc::QuantumComputation> qc = std::make_unique<qc::QuantumComputation>();
+
     if (py::isinstance<py::str>(circ)) {
-        auto&&                                  file1 = circ.cast<std::string>();
-        std::unique_ptr<qc::QuantumComputation> qc    = std::make_unique<qc::QuantumComputation>();
-        std::ofstream                           ofs(filename);
+        auto&& file1 = circ.cast<std::string>();
         qc->import(file1);
-        qc->dump(ofs, qc::Tensor);
     } else if (py::isinstance(circ, QuantumCircuit)) {
-        qc::qiskit::QuantumCircuit::dumpTensorNetwork(circ, filename);
+        qc::qiskit::QuantumCircuit::import(*qc, circ);
     } else if (py::isinstance(circ, pyQasmQobjExperiment)) {
-        qc::qiskit::QasmQobjExperiment::dumpTensorNetwork(circ, filename);
+        qc::qiskit::QasmQobjExperiment::import(*qc, circ);
```

Comment:
Does this make the respective routines in the QFR obsolete? If so, would you mind creating a PR for that?

Reply:
The routines were already removed (or I didn't find them), which is why I changed the lines in the first place :sweat_smile:

Reply:
😅
Good point. Anyway, I like the new solution. Feel free to resolve this conversation.